### ISSUE TITLE
Mirroring challenges removes requirements from challenge.yml

### ISSUE THREAD

**Issue by SleepyHIpster** [CTFd/ctfcli#146]
When using the mirror command on a challenge or across all challenges, the requirements section is removed from the corresponding challenge.yml.

**Comment by ColdHeat** [comment]
Yeah that's definitely not great. PR welcome for this if you have the time.
